include/openthread/tcat.h

```diff
@@ -126,16 +124,16 @@ typedef enum otTcatCommandClass
 } otTcatCommandClass;
 
 /**
- * Represents Advertised Device ID type. (used during TCAT advertisement)
+ * Represents Advertised Device ID type. (Used during TCAT advertisement)
```

Comment:
```suggestion
 * Represents Advertised Device ID type. Used during TCAT advertisement.
```

No need for a parenthetical here